redaxo/src/addons/metainfo/lib/handler/media_handler.php

```diff
@@ -155,6 +155,7 @@ protected function handleSave(array $params, rex_sql $sqlFields)
         if ('post' != rex_request_method() || !isset($params['id'])) {
             return $params;
         }
+        dump(1);
```

Comment:
Debug out

Reply:
Irgendwie warst du in einem alten diff, denn die zeile hatte ich bereits wieder entfernt.